Assets/_Scripts/Gameplay/ReviewDataSO.cs

```diff
@@ -10,6 +10,15 @@ public class ReviewDataSO : ScriptableObject
         public MusicianAppropriatenessFeedback musicianChoiceFeedback;
         public InstrumentFeedback instrumentChoiceFeedback;
         public AffinityFeedback affinityFeedback;
+        public ReviewImageChoices reviewImageChoices;
+        public int stars;
+        public string date;
+        public string caption;
+        public int issueNumber;
+        public int volNumber;
+        public string reviewTitle;
+        public string reviewSubTitle;
+        public Sprite reviewImage;
```

Comment:
Did these need to be added here? They will be fixed per-night if so.

Reply:
Idk if review title == performance title. Ill let @baconeta  decide.